packages/zotero/zotero.pacscript

```diff
@@ -0,0 +1,29 @@
+name="zotero"
+version="5.0.96.2"
+url="https://download.zotero.org/client/release/${version}/Zotero-${version}_linux-x86_64.tar.bz2"
+description="Your personal research assistant"
+hash="653faac4d2e3746dbd0e1c95d095ed8031c38b68183f22ab6708093aba392082"
+maintainer="Falcon Programmer <programmerfalcon@gmail.com>"
```

Comment:
Who is this `Falcon Programmer`? This field should point to @JonBoyleCoding's details

Reply:
```suggestion
maintainer="JonBoyleCoding <programmerfalcon@gmail.com>"
```
Changing it to your GH username will help users find you.